test/tests.py

```diff
@@ -1700,6 +1700,10 @@ def test(self):
                       encoding='utf-8').stdout.splitlines()
               if x.startswith('ATTR ')}
 
+        # access count is added for objects implementing io_memory or transaction
+        # interfaces, but not for any other object, i.e., it is illegal in dml
+        sl.add("access_count")
```

Comment:
I think this is the best path in the short term (for merge technical reasons), but should be improved. I suggest a comment like:
```
# access count is added for objects implementing io_memory or transaction
# interfaces. In DML the name is illegal to avoid a clash in bank objects.
# TODO: we should adjust the test to also extract the set of attributes from
# an empty bank, not only the device object.
```